packages/flutter/test/material/slider_test.dart

```diff
@@ -978,6 +978,63 @@ void main() {
     await tester.pumpAndSettle();
   });
 
+  testWidgets('Tick marks are skipped when they are too dense', (WidgetTester tester) async {
+    double value = 25.0;
+
+    Widget buildSlider({
+      int divisions,
+      TextDirection textDirection,
+    }) {
+      return Directionality(
+        textDirection: textDirection,
+        child: StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return MediaQuery(
+              data: MediaQueryData.fromWindow(window),
+              child: Material(
+                child: Center(
+                  child: Slider(
+                    min: 0.0,
+                    max: 100.0,
+                    divisions: divisions,
+                    value: value,
+                    onChanged: (double newValue) {
```

Comment:
Simplify for this test, since we're not moving the slider:
```
onChanged: (double value) { },
```

Reply:
Done